lineapy/db/db.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, url: str):
         self.session.configure(bind=self.engine)
         Base.metadata.create_all(self.engine)
 
+        # establish alembic db version for migration
+        # https://alembic.sqlalchemy.org/en/latest/cookbook.html#building-an-up-to-date-database-from-scratch
+        from alembic import command
+        from alembic.config import Config
+
+        lp_install_dir = Path(__file__).resolve().parent.parent
+
+        alembic_cfg = Config(lp_install_dir / "alembic.ini")
+        alembic_cfg.set_main_option(
+            "script_location", (lp_install_dir / "_alembic").as_posix()
+        )
+        alembic_cfg.set_main_option("sqlalchemy.url", self.url)
+        command.stamp(alembic_cfg, "head")
```

Comment:
need to check if the alembic table already exists or not. if it does, skip this step. 

also think its safe to add the upgrade step here.